pkg/common/util/config.go

```diff
@@ -50,3 +50,33 @@ func LoadConfig() error {
 	config.CONFIG.AddSource(fSource)
 	return nil
 }
+
+// This function is used  to Load Configarations from a specified location.
```

Comment:
change comment to start with
LoadConfigFromLocation is used to ... 
Also remove double spaces in comment